### PR TITLE
Set unattended-upgrades on Ubuntu hosts

### PR DESCRIPTION
**Pull request by edunham**
Not sure if this is the best way, but http://blog.viraptor.info/post/75262988169/ubuntu-unattended-upgrades-the-salt-way suggests it.

Fixes https://github.com/servo/saltfs/issues/275

r? @aneeshusa

---

This change is [Reviewable](https://reviewable.io/reviews/servo/saltfs/276)

### CODE REVIEW

**Comment by aneeshusa** [comment]
I don't think we actually need the `cmd.wait` here, the `debconf.set` should handle everything. We should check in Vagrant. Also FYI, `cmd.wait` has recently been deprecated, so we shouldn't use it in any new commits (I removed our last remaining uses during the Android refactor).

**Comment by aneeshusa** [comment]
We'll also need a `pkg.installed` state to install the `unattended-upgrades` package, which should require the `debconf.set` state (we can nest them under the same ID).

**Comment by aneeshusa** [comment]
Actually, I  think we'll need to replace the `cmd.wait` state with a `file.managed` state that does the same thing by putting the following into `/etc/apt/apt.conf.d/20auto-upgrades`:
```
APT::Periodic::Update-Package-Lists "1";
APT::Periodic::Unattended-Upgrade "1";
```

**Comment by aneeshusa** [comment]
Do we want the machines to auto-reboot if necessary? I'm assuming not, but if so we'll need to do a little more to activate that feature.

**Comment by edunham** [comment]
@aneeshusa I don't think that auto-rebooting would be a good idea right now. Latest commit implements the other 3 changes you suggested.

**Comment by aneeshusa** [comment]
Didn't see you had updated this, sorry.

This is going to require some in Vagrant testing - I suspect the `unattended-upgrades` package has an associated service we'll need to start (due to #229).

File: common/init.sls [hunk 1]
```diff
@@ -6,10 +6,24 @@ python2:
       - python
 
 {% if grains['os'] == 'Ubuntu' %}
-python2-dev:
+ubuntu-packages:
```
Comment (aneeshusa) [inline common/init.sls]: I'd prefer to keep this state ID as just `python2-dev`, and add a new `pkg.installed` state under the `unattended-upgrades` ID to install the respective package. That lets us keep all of the unattended upgrades states together.

Comment (aneeshusa) [inline common/init.sls]: Also, the new pkg.installed state should `require` the debconf state for ordering purposes.

**Pull request closed without merging**